Cargo.toml

```diff
@@ -152,6 +152,6 @@ ignored = [
 
 # # Uncomment for testing during bones development, if you clone bones adjacent to the jumpy
 # # repo.
-# [patch.'https://github.com/fishfolk/bones']
+[patch.'https://github.com/fishfolk/bones']
```

Comment:
Super minor comment but could we remove this diff / keep this line commented? :) Thanks

Otherwise I think this looks good to merge.

Reply:
fixed this